release_notes.md

```diff
@@ -1,5 +1,9 @@
 ### Version 0.0.19 ###
 
+* Improved the structure of Python samples
+  * If all sampled vertices are scalar then you are returned a dictionary keyed on vertex ID with the values containing a list of primitive values
+  * If any of the sampled vertices are tensors then you are returned a dictionary keyed on a tuple of vertex ID and tensor index. 
+  This makes it much easier to extract all the samples for a given index and for creating a multi-index dataframe  
```

Comment:
Can you be more explicit about the changes you've made to the API for `autocorrelation` and `posterior_samples.get`?

Reply:
yes